src/components/dashboard/nav.tsx

```
@@ -7,15 +7,15 @@ import { redirect } from "next/navigation";
 const fetchOrgs = React.cache(fetchUserOrgs);
 
 const DashboardNav: React.FC<{}> = async () => {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)